src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.SslProvider.cs

```diff
@@ -25,6 +24,8 @@ private static class SslProvider
 
             internal static void SetSslOptions(EasyRequest easy, ClientCertificateOption clientCertOption)
             {
+                easy.SetCurlOption(Interop.Http.CURLoption.CURLOPT_SSLVERSION, (long)Interop.Http.CurlSslVersion.CURL_SSLVERSION_TLSv1); // disable SSLv2/SSLv3
```

Comment:
I went to look this up in the documentation to confirm that this wasn't going to lock it to 1.0.

Might be worth a comment that says that this value enables all TLS 1.x versions and disables SSLv2 and SSLv3.